meson.build

```
@@ -0,0 +1,494 @@
+            cfg_var.set('HAVE_NUMBER_SCENES', '1')
+        endif
+        func_names = [ 'InheritException', 'AcquireExceptionInfo', 'SetImageProperty', 'SetImageExtent', 'AcquireImage', 'GetVirtualPixels', 'ResetImageProfileIterator', 'ResetImageAttributeIterator', 'ResetImagePropertyIterator', 'MagickCoreGenesis', 'SetImageOption', 'BlobToStringInfo', 'OptimizePlusImageLayers', 'OptimizeImageTransparency' ]
+        foreach func_name : func_names
+            if cc.has_function(func_name, prefix : '#include <magick/api.h>', dependencies: magick_dep)
+                cfg_var.set('HAVE_' + func_name.to_upper(), '1')
+            endif
+        endforeach
+        if cc.compiles('#include <magick/api.h>\nColorspaceType colorspace = CMYColorspace;', dependencies: magick_dep)
+            cfg_var.set('HAVE_CMYCOLORSPACE', '1')
+        endif
+        if cc.compiles('#include <magick/api.h>\nColorspaceType colorspace = HCLpColorspace;', dependencies: magick_dep)
+            cfg_var.set('HAVE_HCLPCOLORSPACE', '1')
+        endif
+        # GetImageMagick() takes two args under GM, three under IM
+        if cc.compiles('#include <magick/api.h>\nint main() {(void)GetImageMagick(NULL, 0, NULL);}', dependencies: magick_dep)
+            cfg_var.set('HAVE_GETIMAGEMAGICK3', '1')
+        endif
+    else
+        error('Unsupported libmagick version')
+    endif
+
+    if 'load' in get_option('magick-features')
+        cfg_var.set('ENABLE_MAGICKLOAD', '1')
+    endif
+    if 'save' in get_option('magick-features')
+        cfg_var.set('ENABLE_MAGICKSAVE', '1')
+        if cc.has_function('ImportImagePixels', prefix : '#include <magick/api.h>', dependencies: magick_dep)
+            cfg_var.set('HAVE_IMPORTIMAGEPIXELS', '1')
+        endif
+        if cc.has_function('ImagesToBlob', prefix : '#include <magick/api.h>', dependencies: magick_dep)
+            cfg_var.set('HAVE_IMAGESTOBLOB', '1')
+        endif
+    endif
+endif
+
+cfitsio_dep = dependency('cfitsio', required: get_option('cfitsio'))
+if cfitsio_dep.found()
+    libvips_deps += cfitsio_dep
+    cfg_var.set('HAVE_CFITSIO', '1')
+endif
+
+# quant package we use
+quantisation_package = disabler()
+
+imagequant_dep = dependency('imagequant', required: get_option('imagequant'))
+if imagequant_dep.found()
+    libvips_deps += imagequant_dep
+    cfg_var.set('HAVE_IMAGEQUANT', '1')
+    quantisation_package = imagequant_dep
+endif
+
+# only if libimagequant not found
+quantizr_dep = disabler()
+if not quantisation_package.found()
+    quantizr_dep = dependency('quantizr', required: get_option('quantizr'))
+    if quantizr_dep.found()
+        libvips_deps += quantizr_depquantizr_dep
+        cfg_var.set('HAVE_QUANTIZR', '1')
+        quantisation_package = quantizr_dep
+    endif
+endif
+
+cgif_dep = disabler()
+if quantisation_package.found()
+    cgif_dep = dependency('cgif', required: get_option('cgif'))
+    if quantizr_dep.found()
```

Looks like a copy/paste error.
```suggestion
    if cgif_dep.found()
```

---

Thanks, looks like I made many :sweat: